model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -403,6 +470,33 @@ message Coder {
   repeated string component_coder_ids = 2;
 }
 
+message StandardCoderUrns {
```

Comment:
I believe these are all actually required, but check to be sure.

Missing TIMESTAMP (I think durations are not primitives)